.github/workflows/ci.yml

```diff
@@ -20,6 +20,15 @@ jobs:
     - run: yarn install
     - run: yarn build
 
+    - name: Set Chrome Version
+      run: |
+        CHROMEVER="$(chromedriver --version | cut -d' ' -f2)"
+        echo "Actions ChromeDriver is $CHROMEVER"
+        CONTENTS="$(jq '.tunnelOptions.drivers[0].name = "chrome"' < intern.json)"
+        CONTENTS="$(echo ${CONTENTS} | jq --arg chromever "$CHROMEVER" '.tunnelOptions.drivers[0].version = $chromever')"
+        echo "${CONTENTS}" > intern.json
+        cat intern.json
+
```

Comment:
This script uses the actions environment to determine the correct Chrome version, and writes into Intern.json the version to use. I think intern still downloads it's own copy of ChromeDriver, but at least it gets the tests _running_ :joy:

Reply:
Yak riders, saddle up!